.ci/eclipse-compiler-javac.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -e
+
+ECJ_VERSION="4.7"
+mkdir -p target/classes
+mkdir -p target/eclipse
+
+if [ ! -e ~/.m2/repository/"ecj-$ECJ_VERSION.jar" ]; then
+    wget http://ftp-stud.fht-esslingen.de/pub/Mirrors/eclipse/eclipse/downloads/drops4/R-4.7-201706120950/"ecj-$ECJ_VERSION.jar" -P ~/.m2/repository/
```

Comment:
@romani explained it above.
Repeating here:

the latest 3.13.0 (by the way, do you know why 3.x.x? Currently I used 4.7.0) is not working by default. It says -

`Unable to load annotation processing manager org.eclipse.jdt.internal.compiler.apt.dispatch.BatchAnnotationProcessorManager from classpath.
`
I've found the only old issue - https://bugs.eclipse.org/bugs/show_bug.cgi?id=507370 , but it is already fixed.

However, I found way to get 4.7 version. I used direct link on one of mirrors -
http://ftp-stud.fht-esslingen.de/pub/Mirrors/eclipse/eclipse/downloads/drops4/R-4.7-201706120950/ecj-4.7.jar

Not sure if it ok... But 4.7 works fine.